MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/ComposeContentViewModel.swift

```diff
@@ -228,34 +228,23 @@ public final class ComposeContentViewModel: NSObject, ObservableObject {
         }
 
         // set limit
-        let _configuration: Mastodon.Entity.Instance.Configuration? = {
+        if let configuration: Mastodon.Entity.V2.Instance.Configuration = {
+            var configuration: Mastodon.Entity.V2.Instance.Configuration? = nil
+            context.managedObjectContext.performAndWait {
+                let authentication = authContext.mastodonAuthenticationBox.authentication
+                configuration = authentication.instance(in: context.managedObjectContext)?.configurationV2
```

Comment:
Memo to self: We could look into storing the `configuration`-JSON on disk to avoid the `context.managedObjectContext.performAndWait`-dance

Reply:
Yes definitely, this will be much appreciated, I'll create a ticket for this.